alternator/executor.cc

```diff
@@ -5860,32 +5871,43 @@ static lw_shared_ptr<keyspace_metadata> create_keyspace_metadata(std::string_vie
     }
     auto opts = get_network_topology_options(sp, gossiper, rf);
 
-    // Even if the "tablets" experimental feature is available, we currently
-    // do not enable tablets by default on Alternator tables because LWT is
-    // not yet fully supported with tablets.
-    // The user can override the choice of whether or not to use tablets at
-    // table-creation time by supplying the following tag with a numeric value
-    // (setting the value to 0 means enabling tablets with automatic selection
-    // of the best number of tablets).
+    // Whether to use tablets for the table (actually for the keyspace
+    // of the table) is determined by the parameter `tablets_mode` (which is
+    // determined by `tablets_mode_for_new_keyspaces` config flag), as well as
+    // the presence and the value of a per-table tag below.
+    // Setting the tag with a numeric value will enable a specific initial number
+    // of tablets (setting the value to 0 means enabling tablets with
+    // an automatic selection of the best number of tablets).
     // Setting this tag to any non-numeric value (e.g., an empty string or the
     // word "none") will ask to disable tablets.
-    // If we make this tag a permanent feature, it will get a "system:" prefix -
-    // until then we give it the "experimental:" prefix to not commit to it.
-    static constexpr auto INITIAL_TABLETS_TAG_KEY = "experimental:initial_tablets";
-    // initial_tablets currently defaults to unset, so tablets will not be
-    // used by default on new Alternator tables. Change this initialization
-    // to 0 enable tablets by default, with automatic number of tablets.
+    // When vnodes are asked for by the tag value, but tablets are enforced by config,
+    // throw an exception to the client.
     std::optional<unsigned> initial_tablets;
     if (feat.tablets) {
         auto it = tags_map.find(INITIAL_TABLETS_TAG_KEY);
         if (it != tags_map.end()) {
-            // Tag set. If it's a valid number, use it. If not - e.g., it's
-            // empty or a word like "none", disable tablets by setting
-            // initial_tablets to a disengaged optional.
+            // The per-table tag is set.
             try {
-                initial_tablets = std::stol(tags_map.at(INITIAL_TABLETS_TAG_KEY));
+                // If the tag is a valid number, use it.
+                initial_tablets = std::stol(it->second);
+                elogger.info("Following {} tag, Alternator will attempt to create a keyspace {} with tablets, initially {} of them (0 means default number).", INITIAL_TABLETS_TAG_KEY, keyspace_name, initial_tablets.value());
```

Comment:
I think "info" level is too much, it should be "trace". The thing is, you're not reporting here a rare occurance - you're actually tracing every single use of this tag.